coremltools/models/model.py

```diff
@@ -347,19 +349,15 @@ def __str__(self):
 
     def save(self, filename):
         """
-        Save the model to a .mlmodel format. For MIL program filename is
+        Save the model to a `.mlmodel` format. For MIL program filename is
```

Comment:
Do you want "``.mlmodel``" rather than "`.model`"? If not, what is the difference between single backtick and double backtick?

Reply:
Single back-tic just puts the phrase in italics, which is not what we want. Doc style guides typically call for a filename, or filename extension, to be in code font, which is double-back-tics. We are not consistent about this. But whenever I edit, I try to bring some consistency to it.